owncloudApp/src/main/res/values-hdpi/dims.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You forgot to add the copyright to the file! Don't worry, ask us how to configure it and we will help you 😉